src/components/UserLogin/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { UserLogIn } from '../../interfaces';
+import { googleSignIn, logIn } from '../../auth';
+
+const initialValue: UserLogIn = {
+  email: '',
+  password: '',
+};
+
+const UserLogin: React.FC = () => {
+  const [userLogInfo, setUserLogInfo] = useState<UserLogIn>(initialValue);
+  const navigate = useNavigate();
+
+  const handleSubmit = async (e: React.MouseEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    try {
+      await logIn(userLogInfo.email, userLogInfo.password);
+      navigate('/');
+      console.log('User info: ', userLogInfo);
+    } catch (error) {
+      console.log('Error', error);
+    }
+  };
+
+  const handleGoogleSignIn = async (e: React.MouseEvent<HTMLElement>) => {
+    e.preventDefault();
```

Comment:
nitpick: `e.preventDefault` serve solitamente per prevenire il comportamento del browser quando submitti un form, ma in questo caso non stai submittando un form, stai solo premendo un pulsante, quindi non serve